.pre-commit-config.yaml

```diff
@@ -4,8 +4,8 @@ repos:
     rev: 5.0.4
     hooks:
       - id: flake8
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+  - repo: https://github.com/zhouzaida/isort
+    rev: 5.12.1
```

Comment:
Please try following the fix in MMDetection: https://github.com/open-mmlab/mmdetection/pull/9701